ui/src/authentication/pages/Login/Login.js

```diff
@@ -61,3 +61,14 @@ export default class Login extends Component {
     );
   }
 }
+
+export default class Login extends Component {
```

Comment:
Is this component needed? Putting the Notifications in the LoginPage wouldn't do the job?

Reply:
No because in every rerender the Notifications component would be initialized again losing any information :)